packages/sanity/src/_internal/cli/commands/migration/runMigrationCommand.ts

```diff
@@ -114,18 +118,18 @@ const runMigrationCommand: CliCommandDefinition<CreateFlags> = {
     if (resolvedScripts.length > 1) {
       // todo: consider prompt user about which one to run? note: it's likely a mistake if multiple files resolve to the same name
       throw new Error(
-        `Found multiple migrations for "${id}" in current directory ${candidates
-          .map((candidate) => candidate!.relativePath)
-          .join(', ')}`,
+        `Found multiple migrations for "${id}" in the ${chalk.cyan(migrationsDirectoryPath)} directory: \n - ${candidates
```

Comment:
Not a native english speaker, but could we simplify to `Found multiple migrations for <id> in <migrationsDir>`?

Reply:
I agree. Thanks!